nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/resources/default-grok-patterns.txt

```diff
@@ -0,0 +1,115 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
What's the origin of this file?

Reply:
It if from the standard serialization services, it existed for the GrokReader, I copied it over

Reply:
Ok, so it's a known quantity.